src/hosted-buttons/utils.js

```diff
@@ -77,8 +78,11 @@ const getButtonVariable = (variables: ButtonVariables, key: string): string =>
 export const getHostedButtonDetails: HostedButtonDetailsParams = async ({
   hostedButtonId,
 }) => {
+  const { lang, country } = getLocale();
```

Comment:
I think you can fix the test failure by mocking `getLocale` here: https://github.com/paypal/paypal-checkout-components/blob/88013eaa39a5f54a66f4befffe589c8e121f6945/src/hosted-buttons/utils.test.js#L20-L27

Reply:
Ahhh, thank you so much!

Reply:
This did fix it! I had to add it to index.test.ts as well. 🚀